### ISSUE TITLE
Planter not planting sugarcane in the correct location and continuously breaks and replaces

### ISSUE THREAD

**Issue by Pluto9249** [ldtteam/minecolonies#6409]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.16.4
- MineColonies Version: 0.13.557-ALPHA-universal
- Structurize Version: structurize-0.13.104-ALPHA-universal
- Related Mods and their Versions: N/A

## Expected behavior

The planter working at the plantation would utilize the sand surrounding the pond within its chunks to plant.
## Actual behavior

The first time I made the planter they started planting sugarcane about 10 blocks above the ground. I thought it was a bug with the Fortress version, so we tore it down and rebuilt the medieval spruce version in another area. However, the planter still doesn't plant the sugarcane in the right spot and it cannot grow. The planter walks to this couriers hut about 15 blocks away and keeps planting and immediately harvesting two sugarcane, so nothing is being grown.

Edit: I actually just noticed what might be the problem. The sugarcane she is supposed to be planting is 12 blocks away from the hut. She is planting them 12 X blocks in the OPPOSITE direction, but in the correct Z location. It seems like there is some rotational area going on.

![image](https://user-images.githubusercontent.com/77600115/104870485-a2c67600-5916-11eb-99a3-ef684eeb34a5.png)
You can see the planter hut behind Laura and Laura is continuously placing and breaking these sugarcane without any water being nearby.

## Steps to reproduce the issue

Build a plantation hut, hire a worker, see if they plant the sugarcane in the correction location.
1. 
2. Build plantation hut.
2. 
3. Hire worker, give them sugarcane.
3. 
Planter doesn't plant sugarcane in the correction location.
### Logs

- latest.log: 
- crashlog: 

### Notes

---
## Viewers

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by ravenbuilder934** [comment]
This should only happen when upgrading the plantation iirc

**Comment by Pluto9249** [comment]
> This should only happen when upgrading the plantation iirc

I believe this is happening due to the rotation of the building when you place it. I think it's trying to plant where it's supposed to if we never rotated the building when we used to place the hut using the build tool. Its happening with different types of plantations and at all levels. I noticed this at level 1, then tried to upgrade to 2 and she still plants it at the same spot.
